File: LovelyKumari/BasicProgramming/Problem29.java
import java.util.Scanner;

public class Problem29 {
    //WAP to input a number from user and print its table
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter a number: ");
        int num=sc.nextInt();

        System.out.println("Multiplication table of " + num + ":");
        for(int i=1; i<=10;i++){
            System.out.println(num+ " x " + i + " = " + (num*i));
        }
    }
    
}

File: DheerajKumar/problem29.cpp
//29. WAP to input a number from user and print its table.

#include<iostream>
using namespace std;

void tableOfNumber(int number){
    if(number <= 0){
        return;
    }
    for(int i = 1; i <= 10; i++){
        cout<<number<<" x "<<i<<" = "<<number * i<<"\n";
    }
    return;
}

int main(){
    int number;
    cout<<"Enter the number ->> ";
    cin>>number;

    cout<<"\n";
    tableOfNumber(number);
}
